### ISSUE TITLE
Invalid code for method calls from explicit base contract.

### ISSUE THREAD

**Issue by michael-emmi** [SRI-CSL/solidity#31]
The following code seems valid, but it’s translation is not.
```
$ cat Test.sol && solc-verify.py Test.sol --out . ; grep "A#6" Test.sol.bpl 
// Test.sol
pragma solidity ^0.5.0;

contract A {
    function f() public { }
}

contract B is A {
    function f() public {
        A.f();
    }
}
Error while running verifier, details:
[TRACE] Using prover: /usr/local/bin/z3
Parsing ./Test.sol.bpl
./Test.sol.bpl(34,10): Error: undeclared identifier: A#6
1 name resolution errors detected in ./Test.sol.bpl

	call f#5(A#6, __this, 0);
```

**Comment by dddejan** [comment]
Related to #17

**Comment by dddejan** [comment]
Related to #8

**Comment by michael-emmi** [comment]
Ah yes, this is exactly the same as #8

**Comment by hajduakos** [comment]
Well, not 100% the same, because #8 uses explicit scoping on the current contract and this one uses explicit scoping on the base. But both are fixed in  edf951f. Also, `super.f()` is supported.
